modules/silicon_design/scripts/build/notebooks/cfuplayground/cfuplayground.md

```diff
@@ -0,0 +1,74 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# CFU Playground
+
+
+### Design Space Exploration
+
+```python tags=["parameters"]
+# Vexriscv soft core parameters available for tuning
+bypass            = True
+cfu               = True
+dCacheSize        = 2048
+hardwareDiv       = True
+iCacheSize        = 1024
+mulDiv            = True
+prediction        = "none"
+safe              = True
+singleCycleShift  = True
+singleCycleMulDiv = True
+```
+
+```python
+# Constants 
+CSR_PLUGIN_CONFIG = "mcycle"
+TARGET            = "digilent_arty"
+```
+
+```python
+# Change directory to design space exploration project in CFU-Playground
+%cd /CFU-Playground/proj/dse_template
```

Comment:
do you need to `%cd` so that you can `import dse_framework` or are there other part of the cfu flow dependent on stuff being executed from this location?

Reply:
both are correct: to import as well as execute other parts of the cfu